### PR TITLE
fix: made update invoice on single tap

### PR DESCRIPTION
**Pull request by Aditi-Singh16**
Fixes for #2235 

**Preview**

https://github.com/GaloyMoney/galoy-mobile/assets/74586135/c8560fa3-94a4-4400-91b3-585698a10922

### CODE REVIEW

**Comment by nicolasburtey** [comment]
> Fixes for #2235
> 
> **Preview**
> 
>  2235.mp4

I think the test will have to be updated as well

File: app/screens/receive-bitcoin-screen/receive-btc.tsx [hunk 1]
```diff
@@ -244,21 +244,17 @@ const ReceiveBtc = () => {
...
-
-        <GaloyPrimaryButton
-          {...testProps(LL.ReceiveWrapperScreen.updateInvoice())}
-          title={LL.ReceiveWrapperScreen.updateInvoice()}
-          onPress={() => {
-            setShowMemoInput(false)
-            generatePaymentRequest && generatePaymentRequest()
-          }}
-          disabled={!memo}
-        />
```
Comment (nicolasburtey) [inline app/screens/receive-bitcoin-screen/receive-btc.tsx:261]: should we keep the primaryButton nonetheless

what do you think @sandipndev @UncleSamtoshi ?

**Comment by Aditi-Singh16** [comment]
Is this how it should be? Now user can update note with a single or a double tap, as he/she likes
https://github.com/GaloyMoney/galoy-mobile/assets/74586135/98f284f2-c0b2-4fce-a7fe-aa8bae6f03cd

**Comment by nicolasburtey** [comment]
tests not passing even on android. can you rebase and see if that fixes it?

**Comment by Aditi-Singh16** [comment]
> tests not passing even on android. can you rebase and see if that fixes it?

It passed

**Comment by nicolasburtey** [comment]
closing this stale PR (receive screen has been revamped completely)

**Pull request closed without merging**